src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let modifiedState = Object.assign(state);
```

Comment:
This assignment is redundant here, you don't create new object, just copy link from state to modifiedState